app/src/renderer/components/common/AppMenu.vue

```diff
@@ -8,7 +8,7 @@ menu.app-menu
     part(title='Governance' v-if="config.devMode")
       list-item(to="/proposals" exact @click.native="close" title="Proposals")
     part(title='Stake' v-if="config.devMode")
-      list-item(to="/staking" exact @click.native="close" title="Delegates")
+      list-item(to="/staking" exact @click.native="close" title="Delegate")
```

Comment:
hahaha. we keep switching this back and forth. the reason i changed it to `Delegates` with an 's' is for a couple reasons.

1. Transactions, Proposals, Validators, `Delegates` (nouns) ... but i guess i see how `Delegate` fits with send (verbs)

2. when you click the link `Delegates` what you see is a list of delegates (noun), not a screen for delegating (verb).

this highlights a bigger issue that you brought up in #128 and that i've been thinking about — ideally, our sidebar would use consistent language. actions (verbs) should take place within the context of important nouns (Balances, Transactions, Proposals, Delegates, Validators). your idea to include send as a modal on balances makes a ton of sense to me for this reason. also, why `Delegate`, the action, makes sense as a feature modal of the `Delegates` page. bonding ATOMs is a type of transaction and should have a similar UX to sending.

see also #223

Reply:
Read your reasoning -- let's change it to Delegates.